src/redux/actions/questionsActions/getQuestions.js

```diff
@@ -7,7 +7,7 @@ import { GET_QUESTIONS } from "../actionTypes.js";
 export function getProductQuestions(id) {
   return function(dispatch) {
     return axios
-      .get(`http://3.134.102.30/qa/${id}`)
+      .get(`http://3.134.102.30/qa/${id}?count=100`)
```

Comment:
Interested to hear how you and Mayanne plan on handling this count parameter in the future.